packages/react-components/react-tabster/src/hooks/useOnKeyboardNavigationChange.ts

```diff
@@ -0,0 +1,26 @@
+import { createKeyborg } from 'keyborg';
+import { useEffect, useMemo } from 'react';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import type { KeyborgCallback } from 'keyborg';
+import { useEventCallback } from '@fluentui/react-utilities';
+
+/**
+ * Instantiates [keyborg](https://github.com/microsoft/keyborg) and subscribes to changes
+ * in the keyboard navigation mode.
+ *
+ * @param callback - called every time the keyboard navigation state changes
+ */
+export function useOnKeyboardNavigationChange(callback: (isNavigatingWithKeyboard: boolean) => void) {
+  const { targetDocument } = useFluent();
+  const keyborg = useMemo(() => targetDocument && createKeyborg(targetDocument.defaultView!), [targetDocument]);
```

Comment:
This creates keyborg, but does not dispose it.

Reply:
Yeah good catch - I've added the dispose effect and also a test to make sure that it works